### PR TITLE
Cleanup psq Mnemonics

### PR DESCRIPTION
**Pull request by Cuyler36**
This PR cleans up how psq stores and loads are decompiled. The work was done by @magcius in their fork.

### CODE REVIEW

File: data/languages/ppc_gekko_broadway.slaspec [hunk 1]
```diff
@@ -1910,5 +1879,4 @@ CRM_CR:	cr0		is CRM=128 & cr0	{tmp:4 = zext(cr0) << 28;export tmp;}
 {
 	ea:4 = RA_OR_ZERO + B;
 	dataCacheBlockSetToZeroLocked(ea);
-}
-
+}
```
Comment (aldelaro5) [inline data/languages/ppc_gekko_broadway.slaspec]: can you discard this part?

**Review by aldelaro5 — CHANGES_REQUESTED** [review]
will merge after this

**Review by aldelaro5 — APPROVED** [review]

**Comment by magcius** [comment]
You probably want to take  a look at beholdnec's fork which better improves the ps experience

**Comment by Cuyler36** [comment]
I'll look into it, thanks

**Pull request merged**